clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -764,9 +764,9 @@ AST_POLYMORPHIC_MATCHER(isImplicit,
   return Node.isImplicit();
 }
 
-/// Matches classTemplateSpecializations, templateSpecializationType and
-/// functionDecl that have at least one TemplateArgument matching the given
-/// InnerMatcher.
+/// Matches templateSpecializationTypes, class template specializations,
```

Comment:
The AST matcher changes look correct to me, but you should re-run `clang/docs/tools/dump_ast_matchers.py` to regenerate the HTML documentation.

Reply:
@AaronBallman Already did so :)